examples/ember-cli/todomvc/app/components/todo-item.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+	repo: Ember.inject.service(),
+	tagName: 'li',
+	editing: false,
+	classNameBindings: ['todo.completed', 'editing'],
+
+	actions: {
+		startEditing() {
+			this.get('onStartEdit')();
+			this.set('editing', true);
+			Ember.run.scheduleOnce('afterRender', this, 'focusInput');
```

Comment:
`didRender` will be called whenever this component is rendered or rerendered, while this is a one-time only action. This has to happen specifically after this action and never again.